assets/js/OutputPane.js

```diff
@@ -137,6 +157,83 @@ export default class OutputPane extends React.Component {
     }
   }
 
+  getWindowWidth() {
+      return window.innerWidth
+        || document.documentElement.clientWidth
+        || document.body.clientWidth;
+  }
+
+  resizePaneStart() {
+      this.props.setOverlapping(true);
+      this.props.setFocus();
+  }
+
+  resizePane(e, direction, ref, d) {
+    let offset = this.state.initLeftOffset - d.width;
+    this.setState({
+        leftOffset: offset,
+        resizing: true
+    });
+  }
+
+  resizePaneEnd(e, direction, ref, d) {
+      let width = parseFloat(this.state.width) + ((d.width / this.state.pctBase) * 100) + '%';
+      this.setState({
+          initLeftOffset: this.state.leftOffset,
+          width,
+          resizing: false
+      });
+      this.props.setOverlapping((this.state.leftOffset < 0));
+  }
+
+  reset(libraryState, libraryToggle) {
```

Comment:
reset() may not be the greatest method name -- what is being reset? Relatedly, a comment here wouldn't hurt.